### ISSUE TITLE
Crash while fleeing

### ISSUE THREAD

**Issue by thatmacguy** [michaelchadwick/gemwarrior#53]
```
ROWAD'DE     ::   9 HP
  APATIGER     ::   8 HP

  What do you do?
  [Fight/Attack][Defend][Look][Pass][Run]
  [BATTLE]> r
  You successfully elude apatiger!
  ** POOF **
********************************************************************************
no implicit conversion of Symbol into Integer
######################################################################
Gem Warrior v0.10.8 played for 342 min(s), 41 sec(s), and 486 ms
----------------------------------------------------------------------
Rowad'de   killed 25 monster(s)
           killed 0 boss(es)
           picked up 5 item(s)
           traveled 190 time(s)
           rested 1 time(s)
           died 0 time(s)
######################################################################
Saving game...
Game saved!
```

**Comment by thatmacguy** [comment]
another one:

```
> You wound coraliz for 7 point(s)!
  coraliz attacks you!
  > You are wounded for 15 point(s)!

  ROWAD'DE     ::   9 HP
  CORALIZ      ::   5 HP

  What do you do?
  [Fight/Attack][Defend][Look][Pass][Run]
  [BATTLE]> r
  You successfully elude coraliz!
  ** POOF **
********************************************************************************
no implicit conversion of Symbol into Integer
######################################################################
Gem Warrior v0.10.8 played for 383 min(s), 8 sec(s), and 842 ms
```

**Comment by michaelchadwick** [comment]
I feel like this may have been fixed in [0.11.0](https://github.com/michaelchadwick/gemwarrior/releases/tag/v0.11.0), but I'll leave this open for now until you can confirm.

**Comment by michaelchadwick** [comment]
I'm going to assume this is fixed now.
